onnxruntime/python/onnxruntime_validation.py

```diff
@@ -73,6 +73,9 @@ def validate_build_package_info():
     except ImportError:
         # ORTModule not present
         has_ortmodule = False
+    except AttributeError:
+        # ORTModule not present, this is for Python3.6 compatibility.
+        has_ortmodule = False
```

Comment:
Do we need this file to work in python 3.6? Can onnxruntime even be installed with python3.6 since we should have dropped support already?

If we do need to the compatibility, I would make it very clear in the docstring at the top, then add a per file ignore rule to ignore any “UP” rules (pyupgrade) in https://github.com/microsoft/onnxruntime/blob/ef6f4a4aa1dbf9278379e630055a3075e5cda60e/pyproject.toml#L87C1-L87C1

Reply:
Yeah, you guys are right. And I am also aware that we are dropping python3.6 support. I was kind of asking for some feedbacks implicitly, if we can support without much effort without commitment. :)

Reply:
Considering this is not the only place python3.6 cannot run, let us see whether customers can upgrade the envs...  Thanks you for all the feedbacks.